docs/getting_started/tutorial.rst

```diff
@@ -68,23 +66,25 @@ Use pip, which is packaged with Python, to install Wagtail and its dependencies:
 Generate your site
 ~~~~~~~~~~~~~~~~~~
 
-.. code-block:: console
-
-   $ wagtail start mysite
-   $ cd mysite
-
 Wagtail provides a ``start`` command similar to ``django-admin startproject``.
 Running ``wagtail start mysite`` in your project will generate a new ``mysite`` folder with a few Wagtail-specific extras, including
 the required project settings,
 a "home" app with a blank ``HomePage`` model
 and basic templates
 and a sample "search" app.
 
+Because the folder ``mysite`` was already created by ``venv``, run ``wagtail start`` with an additional argument to specify the destination directory:
+
+.. code-block:: console
+
+   $ wagtail start mysite mysite
```

Comment:
It would be pretty nice if this could, for clarity, be something like 
```$ wagtail start mysite --directory mysite```

Or if, upon finding that a `mysite` directory already existed, it would ask if you wanted to use that directory anyway.

Reply:
(I'm aware that this matches the Django equivalent. But still.)